packages/mobile/routes/dashboard/wallet/drawers/AccountSwitcherDrawer.svelte

```diff
@@ -0,0 +1,44 @@
+<script lang="typescript">
+    import { localize } from '@core/i18n'
+    import { BASE_TOKEN } from '@core/network'
+    import { activeProfile, visibleActiveAccounts } from '@core/profile'
+    import { sumBalanceForAccounts } from '@core/account'
+    import { formatTokenAmountBestMatch } from '@core/wallet'
+    import features from '../../../../features/features'
+    import { AccountSwitcherMenuItem, Drawer } from '../../../../../mobile/components'
+    import { HR, Icon, Text, TextType } from 'shared/components'
+
+    export let onClose: () => unknown = () => {}
+
+    $: totalBalance = sumBalanceForAccounts($visibleActiveAccounts)
+</script>
+
+<Drawer {onClose} title={localize('general.accounts')}>
+    <div class="flex flex-col w-full">
+        <account-list class="accounts flex flex-col scrollable-y">
+            {#each $visibleActiveAccounts as account}
+                <AccountSwitcherMenuItem id="account-{account.index}" {account} onClick={onClose} />
+            {/each}
+        </account-list>
+        <HR />
+        <div class="flex flex-row-reverse h-16">
+            <Text classes="opacity-50 m-auto w-full text-right" type={TextType.h5}>
+                {localize('general.total', {
+                    values: {
+                        balance: formatTokenAmountBestMatch(totalBalance, BASE_TOKEN[$activeProfile.networkProtocol]),
+                    },
+                })}
+            </Text>
+            {#if features?.wallet?.createAccount?.enabled}
+                <button class="h-full" on:click={() => {}}>
```

Comment:
will this onclick be implemented in another step?

Reply:
Yep, it's just a placeholder